src/main/scala/org/scalajs/tools/tsimporter/parser/TSDefParser.scala

```diff
@@ -297,7 +297,8 @@ class TSDefParser extends StdTokenParsers with ImplicitConversions {
     | "public" ^^^ Modifier.Public
     | "readonly" ^^^ Modifier.ReadOnly
     | "protected" ^^^ Modifier.Protected
-  )
+    | lexical.Identifier("abstract") ^^^ Modifier.Abstract
+    )
```

Comment:
The formatting for the `)` was broken here (there should be only 2 spaces before it).